CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### pg_auto_failover v1.0.1 (May 6, 2019) ###
+
+* Fix a problem where the service was not restarted when shutdown (#2)
+* Clarify name in background workers for the monitor (#3)
+* Show full version number in `pg_autoctl version` (#4)
+
```

Comment:
From https://github.com/citusdata/pg_auto_failover/milestone/1?closed=1, let's also add:

 * warn the user when the primary node is not running pg_autoctl as a service while initializing the secondary
 * improve documentation